synfig-studio/src/gui/states/state_eyedrop.cpp

```diff
@@ -145,6 +145,12 @@ StateEyedrop_Context::event_workarea_mouse_button_down_handler(const Smach::even
 	if(event.button==BUTTON_LEFT)
 	{
 		Color color(canvas_view->get_canvas()->get_context(canvas_view->get_context_params()).get_color(event.pos));
+        if(event.modifier&GDK_CONTROL_MASK)
```

Comment:
Ok, looks good, but 2 notices:
1) We are using tabs for whole project, so please use tabs here for indentation;
2) I think code can be optimized more. For example:
```
Color color(canvas_view->get_canvas()->get_context(canvas_view->get_context_params()).get_color(event.pos));

if ((event.modifier & GDK_CONTROL_MASK) == GDK_CONTROL_MASK) {
	synfigapp::Main::set_fill_color(color);
} else {
	synfigapp::Main::set_outline_color(color);
}
studio::App::dialog_color->set_color(color);

return Smach::RESULT_ACCEPT;
```
in this way you can clearly see which part is really changing.

Reply:
So I have fixed.